packages/adapters/ethers/src/client.ts

```diff
@@ -409,6 +409,16 @@ export class EthersAdapter {
         )
       },
 
+      grantPermissions: async params => {
+        const provider = ProviderUtil.getProvider<Provider>('eip155')
```

Comment:
We should use contants:
```suggestion
        const provider = ProviderUtil.getProvider<Provider>(CommonConstantsUtil.CHAIN.EVM)
```